shoop/themes/classic_gray/templates/classic_gray/shoop/front/includes/language_changer.jinja

```diff
@@ -0,0 +1,19 @@
+{% if LANGUAGES|length > 1 %}
+<form method="post" action="/set-language/" class="hidden">
+    {% csrf_token %}
```

Comment:
`/set-language/` should be CSRF exempt, imho, to avoid creating a session where one is not needed.